test/compile-error-test.bats.template

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+# Compile error test
+# 
+# The test case here feeds a malformed .ddl into ddlog's print command and compares whether it produces an expected error.
+
+source bats-template.bash  # for $TESTDIR, $it, etc.
+
+# some preconditions
+setup() {
+    [ -e "$TESTDIR" ]
+    expectedError="$TESTDIR"/compile-error.expected
+    [ -e "$expectedError" ]
+    actualError=${expectedError%.expected}.actual
+}
+
+# check if input produces a compile error
+@test "$it compiles input" {
+    ! ddlog compile "$TESTDIR"/input.ddl >/dev/null 2>"$actualError"
```

Comment:
I think it's a better design to do all the checks in SemanticChecker and not have any checks or error messages in the compiler. The test in its current form can't distinguish where the error comes from.

Also please update the comments when cloning code/scripts. It says the test is doing print.